docs/docsite/rst/user_guide/playbooks_variables.rst

```diff
@@ -171,44 +197,44 @@ Using the dot notation::
 Transforming variables with Jinja2 filters
 ==========================================
 
-Jinja2 filters let you transform the value of a variable within a template expression. For example, the ``capitalize`` filter capitalizes any value passed to it; the ``to_yaml`` and ``to_json`` filters change the format of your variable values. Jinja2 includes many `built-in filters <http://jinja.pocoo.org/docs/templates/#builtin-filters>`_ and Ansible supplies many more filters. See :ref:`playbooks_filters` for examples.
+Jinja2 filters let you transform the value of a variable within a template expression. For example, the ``capitalize`` filter capitalizes any value passed to it; the ``to_yaml`` and ``to_json`` filters change the format of your variable values. Jinja2 includes many `built-in filters <http://jinja.pocoo.org/docs/templates/#builtin-filters>`_ and Ansible supplies many more filters. To know of more examples of filters, see :ref:`playbooks_filters`.
 
 .. _setting_variables:
 
-Where to set variables
-======================
+Where do I define the variables?
```

Comment:
Does the IBM Style Guide recommend using questions in headings? I thought that was, um, frowned upon.